ReactWindows/ReactNative.Core/Bridge/IReactDelegateFactory.cs

```diff
@@ -21,8 +21,9 @@ public interface IReactDelegateFactory
         /// </summary>
         /// <param name="nativeModule">The native module instance.</param>
         /// <param name="method">The method.</param>
+        /// <param name="genericDelegate">The delegate used to avoid expensive reflection invocation.</param>
         /// <returns>The invocation delegate.</returns>
-        Action<INativeModule, IReactInstance, JArray> Create(INativeModule nativeModule, MethodInfo method);
+        Action<INativeModule, IReactInstance, JArray> Create(INativeModule nativeModule, MethodInfo method, IGenericDelegate genericDelegate);
```

Comment:
>Create [](start = 54, length = 6)

Let's eliminate the IGenericDelegate parameter and make it an implementation details of the IReactDelegateFactory.